### PR TITLE
bitwig-studio5: init at 5.0

### PR DESCRIPTION
**Pull request by Philipp-M**
###### Description of changes

This is basically a copy of the `bitwig-studio4` package, it was enough to just change the version to 5.0 (and correct the hash obviously).

It was a painless upgrade, so far all my projects of `bitwig-studio4` are working with it out of the box.

###### Things done

Copied `bitwig-studio4` to `bitwig-studio5`, changed the version and set it to the default `bitwig-studio` package.

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix
Comment (tomcur) [inline pkgs/top-level/all-packages.nix:29946]: Remove this newline to keep the listing of Bitwig versions consistent.

**Comment by tomcur** [comment]
Thank you, tested and works. Lgtm modulo the list consistency in all-packages.nix :+1:

**Comment by Philipp-M** [comment]
Ok, yeah every project worked, but actual "saving as" was broken. I had to add a few dependencies to solve this (pango, atk, harfbuzz).

**Review by bfortz — APPROVED** [review]
Works like a charm!

**Review by tomcur — APPROVED** [review]

**Comment by RobbieGM** [comment]
Is this ready to merge?

**Comment by Philipp-M** [comment]
I guess so, if nobody else has any issues, I haven't noticed any further.

**Comment by bfortz** [comment]
> Is this ready to merge? 

Yes! Please go ahead.

**Comment by RobbieGM** [comment]
> > Is this ready to merge?
> 
> Yes! Please go ahead.

I'd love to but I don't have permission; I'm not a maintainer.

**Comment by polygon** [comment]
Changes match with what I did for the Beta releases of Bitwig 5 and it worked flawlessly for weeks now. Looks good to me.

**Comment by RobbieGM** [comment]
Could somebody with permission to merge this go ahead and do so?

**Comment by RobbieGM** [comment]
Actually I should mention that I'm having trouble using yabridge'd plugins with this change. It might be worth checking if this is a problem in the package, though it could be a yabridge problem too:
```com.bitwig.base.io.Jmb: com.bitwig.flt.library.metadata.reader.exception.CouldNotReadMetadataException: could not read metadata: com.bitwig.flt.library.metadata.reader.exception.CouldNotReadMetadataException: could not read metadata: Failed to load VST 3 plug-in /home/unfree/.vst3/yabridge/Kontakt.vst3: /usr/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.36' not found (required by /nix/store/9rk5pzhnw4sd0x25nwvi2cj7jwmc48k5-gcc-12.3.0-lib/lib64/libstdc++.so.6)```
To reproduce:
1. `nix shell github:Philipp-M/nixpkgs/bitwig-studio5#bitwig-studio5 --impure`
2. `bitwig-studio`, and load some plugins with yabridge
My nixos channel is unstable. `ldd --version` is 2.37.0, using latest yabridge (5.0.5). To anyone following this thread, please let me know whether or not you encounter this error too.

**Comment by Philipp-M** [comment]
I don't think it's a yabridge nor a bitwig-studio issue, I'm not having any issues with windows VSTs in bitwig.

Have you tried cleaning/updating the VSTs (something like `yabridgectl sync -pf` I think)?

For me all the VSTs are linked with glibc 2.37.

**Comment by RobbieGM** [comment]
The creator of yabridge suggested it might be an issue with the version of nixpkgs where Bitwig is compiled from (your fork) being behind the one where yabridge is from (very recently updated from nixpkgs-unstable). So this problem might go away when you merge. But also, after fiddling around randomly with some packages, stuff started working again, albeit with the same error messages harmlessly in the background.

**Comment by stelcodes** [comment]
FWIW I just freshly set up wine and yabridge after installing Bitwig 5 from this derivation and I'm having no issues installing and using Windows VSTs.

**Pull request merged**
